Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'materialize-sass'
 gem 'font-awesome-sass'
 gem 'react-rails'
 gem 'rails_admin', '~> 1.0'
+gem 'pry-rails'
```

Comment:
just noticed this, its cool that you added this, but can you move it in the 'development' group so it doesnt load for everything -

also we already have 'byebug' which is pretty much the same thing, so you shouldnt technically need this.